test/de.cau.cs.kieler.klighd.piccolo.test/src/de/cau/cs/kieler/klighd/piccolo/test/highlightedEdgeToForeground/HighlightedEdgeToForegroundTest.java

```diff
@@ -89,58 +89,64 @@ public void configure(final Binder binder) {
         new ContextViewer(shell).setModel(viewContext, true);
 
         viewContext.update(null);
-        new LightDiagramLayoutConfig(viewContext)
-            .animate(false)
-            .zoomStyle(ZoomStyle.ZOOM_TO_ACTUAL_SIZE)
-            .performLayout();
+        new LightDiagramLayoutConfig(viewContext).animate(false)
+                .zoomStyle(ZoomStyle.ZOOM_TO_ACTUAL_SIZE).performLayout();
```

Comment:
Where is this better readable than the previous version? I would leave it as it was before.